### DISCUSSION TITLE
Heater status and cost estimate.

### DISCUSSION THREAD

**Discussion by wyliecoyoteuk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#145] [Ideas]
I notice that the heater status only tells you whether it is enabled, not its actual state. (this is also true of the Bestway and LayZspa apps)
i.e. the heater goes to set temp then cycles up and down so that:
On the actual pump, it shows 
a) green when enabled but inactive, and
b) red when enabled and active.
Is that information available on the data bus?
Is the cost estimate based on b or a+b?
Once a well insulated tub reaches set temp +1C (or +2C for older pumps) it drops to set temp -1 (or -2), which means that it is only actually using electricity about half the time when enabled.
Also it would be nice if the web interface could display actual status as well as enablement.

**Comment by visualapproach** [comment]
The web interface is showing a green or red slider just like the pump display. It only count the big money when red. (4 wire interface has some improvements to be made. Edit: it was actually the same when in control)
![image](https://user-images.githubusercontent.com/22987250/141160961-2fc851e7-c2e0-489d-8deb-ae1c7be22c82.png)

**Reply by wyliecoyoteuk** [reply]
Neat. I didn't realise that. Thanks.
